src/components/Main/MainView/MessageInput/DropdownSuggester.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <div :class="$style.container">
+    <div
+      :class="{
+        [$style.item]: true,
+        [$style.selected]: index === currentIndex
+      }"
+      v-for="(candidate, index) in candidatesWithId"
+      :key="candidate"
+      @click="onClick(index)"
+    >
+      <template v-if="candidate.isUser">
+        <user-icon :user-id="candidate.userId" :size="24" />
+        <online-indicator
+          :class="$style.indicator"
+          :user-id="candidate.userId"
+          :size="8"
+        />
+        <div :class="$style.name">
+          {{ candidate.word }}
+        </div>
+      </template>
+      <template v-else :class="$style.name">
+        {{ candidate.word }}
+      </template>
```

Comment:
```suggestion
        <div :class="$style.name">
          {{ candidate.word }}
        </div>
      </template>
      <template v-else>
        {{ candidate.word }}
      </template>
```
`template`にはつけても効果ないよ